config/routes.rb

```diff
@@ -1,3 +1,22 @@
 Rails.application.routes.draw do
+  # Routes for the Movie resource:
+
+  # CREATE
+  match("/insert_movie", { :controller => "movies", :action => "create", :via => "post"})
+          
+  # READ
+  match("/movies", { :controller => "movies", :action => "index", :via => "get"})
+  
+  match("/movies/:the_movie_id", { :controller => "movies", :action => "show", :via => "get"})
```

Comment:
@jelaniwoods @pmckernin @logandsprice

Didn't we discuss at some point naming these flexible segments, and also inputs, in a more descriptive manner? I forgot where we came down.

At present I am leaning towards being as descriptive as we can / nipping as much confusing in the bud as we possibly can. Of course, we also don't want them to be 80 characters long.

What do you think about `/movies/:id_from_path`?

And in forms, we can adopt the convention of naming inputs something `_from_query`, e.g. `<input name="title_from_query">`?

Then, the `params` hash will look like

```ruby
{ "id_from_path" => 42, "title_from_query" => "The Shawshank Redemption" }
```

Reply:
@raghubetina I think previously we had something like `/movies/:route_movie_id` and the query string were `qs_movie_title`. Those weren't what the assignments were like, so I made an update to keep it more consistent to what the students have been doing for class. 

I do like your suggestion though! I will make the change.